pages/profile/index.js

```diff
@@ -28,6 +28,10 @@ Profile.propTypes = {
   programmingLanguages: string.isRequired,
 };
 
+Profile.defaultProps = {
+  branchOfService: '',
+};
```

Comment:
Why make this change?

Reply:
<img width="1628" alt="Screen Shot 2020-10-05 at 8 04 49 PM" src="https://user-images.githubusercontent.com/17136024/95144401-968f2200-0746-11eb-9392-2b4ea7816f20.png">
I think only veteran folks have `branchOfService` associated with their account yet since it was required I saw an error in the console

Reply:
Ah, go ahead and undo this change here and I'll make a ticket for the fix

The fix being: Make them _all_ optional. Default to `undefined`. If undefined, the field in the table on profile shouldnt render.

Reply:
made: https://github.com/OperationCode/front-end/issues/1269

Reply:
got it